src/style.css

```diff
@@ -1,3 +1,64 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+  align-items: center;
+}
+
+.logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+  margin-top: 15px;
+  margin-bottom: 10px;
+  text-decoration: none;
+  text-align: center;
+}
+
+.nav_link {
+  display: block;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  margin-left: 20px;
+  padding: 22.5px 10px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  /* I don't now how I can do this */
+  content: '';
+  width: 100%;
+  height: 10px;
+  background-color: #00acdc;
+  border-radius: 30px;
+  text-decoration: underline;
```

Comment:
The `.is-active::after` pseudo-element is intended to add a visual indicator for active links, but the implementation is incomplete. Consider positioning the pseudo-element correctly using properties like `position`, `bottom`, and `left` to ensure it appears as intended.